docs/reference-architectures/virtual-machines-linux/single-vm.md

```diff
@@ -15,25 +15,25 @@ pnp.series.prev: ./index
 
 # Run a Linux VM on Azure
 
-This reference architecture shows a set of proven practices for running a Linux virtual machine (VM) on Azure. It includes recommendations for provisioning the VM along with networking and storage components. This architecture can be used to run a single instance, and is the basis for more complex architectures such as n-tier applications. [**Deploy this solution**.](#deploy-the-solution)
+This reference architecture follows a set of proven practices for running a Linux virtual machine (VM) on Azure. It includes recommendations for provisioning the VM along with networking and storage components. This architecture can be used to run a single VM instance, and is the basis for more complex architectures such as N-tier applications. [**Deploy this solution.**](#deploy-the-solution)
 
 ![[0]][0]
 
-*Download a [Visio file][visio-download] of this architecture.*
+*Download a [Visio file][visio-download] that contains this architecture diagram.*
 
 ## Architecture
 
-Provisioning a VM in Azure involves more moving parts than just the VM itself. There are compute, networking, and storage elements that you need to consider.
+Provisioning a VM in Azure involves more components than only the VM. There are compute, networking, and storage components that you need to consider.
```

Comment:
Agree with changing "moving parts" here. To my ear "only the VM" sounds more awkward than "just the VM" - ? Might just be me 😸

Reply:
Now that I re-read it, I don't think we need either word. :-)

Reply:
sounds good!